includes/admin/shortcodes/shortcode-give-donor-wall.php

```diff
@@ -160,7 +160,7 @@ public function define_fields() {
 			),
 			array(
 				'type'        => 'listbox',
-				'name'        => 'show_time',
```

Comment:
@JoshuaHungDinh existing shortcode properties can not be renamed. If it is necessary then provide backward compatibility for the legacy property. This can cause an unexpected output with PHP warnings. The shortcode saves in the database and it is the possible website still have the legacy property.